### PR TITLE
DashboardScene: Enable scene tracking information

### PR DESCRIPTION
**Pull request by dprokop**
Less invasive alternative to https://github.com/grafana/grafana/pull/78719

Closes https://github.com/grafana/grafana/pull/78719

### CODE REVIEW

File: packages/grafana-runtime/src/analytics/utils.ts [hunk 1]
```diff
@@ -43,14 +43,17 @@ export const reportPageview = () => {
...
-      interactionName,
-      properties,
+export const reportInteraction = (interactionName: string, properties?: Record<string, unknown>, meta?: {}) => {
+  getEchoSrv().addEvent<InteractionEchoEvent>(
+    {
+      type: EchoEventType.Interaction,
+      payload: {
+        interactionName,
+        properties,
+      },
```
Comment (ivanortegaalba) [inline packages/grafana-runtime/src/analytics/utils.ts:53]: Why do we need an extra prop `meta`? Can we just add it to the `properties`?

File: public/app/features/dashboard-scene/utils/interactions.ts [hunk 2]
```diff
@@ -137,12 +139,22 @@ export const DashboardInteractions = {
...
+    };
+  },
 };
 
 const reportDashboardInteraction: typeof reportInteraction = (name, properties) => {
+  const meta = isScenesContextSet ? { scenesView: true } : undefined;
+
   if (properties) {
-    reportInteraction(`dashboards_${name}`, properties);
+    reportInteraction(`dashboards_${name}`, properties, meta);
```
Comment (ivanortegaalba) [inline public/app/features/dashboard-scene/utils/interactions.ts:156]: Why do we need the extra param?
```suggestion
    reportInteraction(`dashboards_${name}`, {...properties, ...meta});
```

File: packages/grafana-runtime/src/analytics/utils.ts (hunk 1 above)
Comment (dprokop) [reply packages/grafana-runtime/src/analytics/utils.ts]: Yeah, you might be right setting this as a property is the way to go. Pushed an update

**Review by ivanortegaalba — APPROVED** [review]
I love it! LGTM! 🚀

File: public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts [hunk 3]
```diff
@@ -67,7 +68,10 @@ export function transformSaveModelToScene(rsp: DashboardDTO): DashboardScene {
     autoMigrateOldPanels: false,
   });
 
-  return createDashboardSceneFromDashboardModel(oldModel);
+  const scene = createDashboardSceneFromDashboardModel(oldModel);
+  scene.state.$behaviors?.push(registerDashboardSceneTracking(oldModel, rsp.dashboard.version));
```
Comment (torkelo) [inline public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts:72]: What's the reason this behavior is added here and not inside createDashboardSceneFromDashboardModel?

**Pull request merged**